src/SprykerEco/Zed/UnzerGui/UnzerGuiConfig.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui;
+
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+
+class UnzerGuiConfig extends AbstractBundleConfig
+{
+ /**
+  * @var int
+  */
+    protected const UNZER_MERCHANT_SUGGESTION_LIMIT = 20;
+
+    /**
+     * @var array<int, string>
+     */
+    protected const UNZER_CREDENTIALS_TYPES = [
```

Comment:
```suggestion
    protected const UNZER_CREDENTIAL_TYPES = [
```

according to method naming below
or vice versa